modular_skyrat/modules/goofsec/code/sec_clothing_overrides.dm

```diff
@@ -480,3 +490,99 @@
 
 /obj/item/pda/heads/hos
 	greyscale_colors = "#2B356D#1E1E1E"
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////
+//A bunch of Re-overrides so that admins can keep using some redsec stuff; not all of them have this though!
+
+//Eyes//
+/obj/item/clothing/glasses/hud/security/redsec
+	icon = 'icons/obj/clothing/glasses.dmi'
+	worn_icon = 'icons/mob/clothing/eyes.dmi'
+	icon_state = "securityhud"
+	glass_colour_type = /datum/client_colour/glass_colour/red
+
+/obj/item/clothing/glasses/hud/security/sunglasses/redsec
+	icon = 'icons/obj/clothing/glasses.dmi'
+	worn_icon = 'icons/mob/clothing/eyes.dmi'
+	icon_state = "sunhudsec"
+	glass_colour_type = /datum/client_colour/glass_colour/darkred
+	current_skin = "sunhudsec"	//prevents reskinning; a bit hacky to say its already reskinned but its better than a code rewrite
```

Comment:
this is not the correct way to go about this

Reply:
The other solution is to set Unique Reskin to False and set the reskin list to Null, but means a lot more code thats also fairly copy-paste-y